### ISSUE TITLE
[FEATURE] Launch parameter feature to start a game by ID

### ISSUE THREAD

**Issue by liamharper2453** [mariotaku/moonlight-tv#200]
**Is your feature request related to a problem? Please describe.**
Currently, as part of https://github.com/liamharper2453/MoonlightTVLauncher I am launching specific games by asking users to specify an index parameter. This index parameter simply simulates pressing the right button on your remote x amount of times to get to your game. It would be great if instead of this we could launch Moonlight TV with a parameter being passed in that would auto-start a game.

**Describe the solution you'd like**
A parameter should be available for the Moonlight TV application that when passed in can automatically open a game. This simply could be an index of the game to be opened or some sort of other identifier.

**Describe alternatives you've considered**
See my hacky alternative way of doing this in https://github.com/liamharper2453/MoonlightTVLauncher/blob/master/MoonlightTVLauncher/Scripts/MoonlightTVAutoConnect.py

**Additional context**
N/A

**Comment by mariotaku** [comment]
`host_uuid` and `host_app_id` will be added. If you launch with those parameters, moonlight will start with the specified host and game.

**Comment by mariotaku** [comment]
Since the feature is added in recent snapshots, you can try launch the app in [this way](https://github.com/supersaiyanmode/PyWebOSTV#application-controls)

And the proper launch param should be:

```python
app = ApplicationControl(client)
moonlight = [x for x in apps if "com.limelight.webos" in x["id"]][0]
app.launch(moonlight, params={'host_uuid': '...', 'host_app_id': ...})
```

**Comment by mariotaku** [comment]
Hi, it's available in v1.5.4. Please let me know if it's not working.
